geps/gep-713/index.md

```diff
@@ -1,1157 +1,1082 @@
+
+##### Targetability (“where”)
 
-## Background and concepts
+Metaresources specify one or more target resources or specific sections of resources, whose behavior the metaresource intends to augment.
 
-When designing Gateway API, one of the things we’ve found is that we often need to be
-able change the behavior of objects without being able to make changes to the spec
-of those objects. Sometimes, this is because we can’t change the spec of the object
-to hold the information we need ( ReferenceGrant, from
-[GEP-709](https://gateway-api.sigs.k8s.io/geps/gep-709/), affecting Secrets
-and Services is an example, as is Direct Policy Attachment), and sometimes it’s
-because we want the behavior change to flow across multiple objects
-(this is what Inherited Policy Attachment is for).
+##### Semantics (“why”)
 
-To put this another way, sometimes we need ways to be able to affect how an object
-is interpreted in the API, without representing the description of those effects
-inside the spec of the object.
+Targeting a resource (or section of a resource) must be interpreted within a given semantics that is proper to the metaresource kind.
 
-This document describes the ways we design objects to meet these two use cases,
-and why you might choose one or the other.
+Two different metaresource kinds that allow targeting resources of a same a given kind X may have very different semantics, not only because the purpose of the two metaresource kinds differ, but because the mechanics of calculating and applying the augmented behavior for X can as well be very different between metaresource kinds.
 
-We use the term “metaresource” to describe the class of objects that _only_ augment
-the behavior of another Kubernetes object, regardless of what they are targeting.
+Often, the semantics of a metaresource is tightly coupled to the relationships and connections a target has with other kinds of objects, typically organized in a hierarchy of nested contexts. In this sense, targeting a given resource kind may have the semantics of spanning effect across yet other objects to which the target is related.
 
-“Meta” here is used in its Greek sense of “more comprehensive”
-or “transcending”, and “resource” rather than “object” because “metaresource”
-is more pronounceable than “metaobject”. Additionally, a single word is better
-than a phrase like “wrapper object” or “wrapper resource” overall, although both
-of those terms are effectively synonymous with “metaresource”.
+##### Mergeability (“how”)
 
-A "Policy Attachment" is a metaresource that affects the fields in existing objects
-(like Gateway or Routes), or influences the configuration that's generated in an
-underlying data plane.
+Metaresources specify so-called *merge strategies* that dictate how multiple instances of the metaresource affecting a same resource (or section of a resource) should be dealt with in terms of calculating and enforcing a single set of rules specified by the metaresources.
```

Comment:
```suggestion
Metaresources can also specify so-called *merge strategies* that dictate how multiple instances of the metaresource affecting the same resource (or section of a resource) should be handled.
```

Reply:
I like the simplification of "[…] should be handled."

Not sure about "can also" though. In my view, all metaresources specify a merge strategy, even if implicitly (by not discriminating any merge strategy and therefore applying the default merge strategy of no merges at all).

Maybe "specify" is not the word, but I would like to be clear that _mergeability_ is a core property of metaresources, even when that merge strategy is no-merge.

Reply:
Ok, I didn't understand that there was an implicit default.  Did I miss it or is it not yet stated?